lib/form/js/repeatable.js

```diff
@@ -0,0 +1,266 @@
+/* eslint-env browser */
+/* global allPlugins, allInputs, allSettings, allIDs, allRepeatables, validate */
+
+(function formRepeatable() {
+  'use strict';
+
+  // Splits the string and pops the last element
+  var splitPop = function pop(input, splitter) {
+    var blocks = input.split(splitter);
+    blocks.pop();
+    blocks = blocks.join(splitter);
+
+    return blocks;
+  };
+
+  // Iterates through every instance of input-plugins
+  Object.keys(allSettings).forEach(function repeatableHandler(input) {
+    var idAdd = input + '--add',
+        idDelete = input + '--delete',
+        deleteHandler,
+        addHandler,
+        addButton;
+
+    deleteHandler = function deleteEventHandler(e) {
+      var node = e.currentTarget.parentNode,
+          siblingNode = node.nextElementSibling,
+          splitChild,
+          nodeId,
+          container,
+          name,
+          childNodes = node.querySelectorAll('*'),
+          parent = node.parentNode,
+          deleteNode;
+
+      // update allInputs and allIds;
+      childNodes.forEach(function deleteIDandInput(child) {
+        if (allInputs.hasOwnProperty(child.id)) {
+          delete allInputs[child.id];
+        }
+        if (allIDs.hasOwnProperty(child.id)) {
+          delete allIDs[child.id];
+        }
+      });
+
+      parent.removeChild(node);
+      allRepeatables[input].length--;
+
+      deleteNode = function deleteElement(key) {
```

Comment:
It _appears_ as if this function is created inside the `deleteHandler` function, and I'm fairly confident that's not your intention. Please make sure all of your parenthesis are in place